source/installation-guide/installing-wazuh-manager/linux/opensuse/wazuh_server_sources_opensuse.rst

```diff
@@ -104,15 +104,15 @@ To check the API service status use the following command:
 
     .. code-block:: console
 
-      # systemctl status wazuh-api
+      # systemctl status wazuh-manager
```

Comment:
Do we need this code? It says to check the API service status, there is not API service. Please review rest of the documentation for API service references.

Reply:
No, this is no longer required. As you can see in the installation guides for other platforms, this fragment must be removed. I will remove it from this page as well and review if it is present in any other page.